src/Navbar/Navbar.vue

```diff
@@ -8,23 +8,25 @@
                tabindex="0"
                v-for="item in items"
                :key="item"
-               @focus="itemSelect($event, 'menu', item)"
-               @blur="itemDeselect($event, 'menu')"
-               @mouseover="itemSelect($event, 'menu', item)"
-               @mouseleave="itemDeselect($event, 'menu')">
+               @focus="itemActivate($event, item)"
+               @blur="itemDeselect({delay: 250})"
+               @touchstart="itemClick($event, item)"
+               @mouseover="itemActivate($event, item)"
+               @mouseleave="itemDeselect({delay: 250})">
             {{item}}
           </div>
         </div>
         <div class="cta"></div>
       </div>
       <transition name="dropdown">
-        <div tabindex="0"
-             @focus="itemSelect($event, 'dropdown', itemSelected)"
-             @blur="itemDeselect($event, 'dropdown')"
-             @mouseover="itemSelect($event, 'dropdown', itemSelected)"
-             @mouseleave="itemDeselect($event, 'dropdown')"
-             class="dropdown"
-             v-if="!!dropdown.visible">
+        <div class="dropdown"
+             :tabindex="dropdown.tabindex"
+             @click="dropdownClick($event)"
+             @blur="dropdownBlur()"
+             @focus="itemSelect(selected)"
+             @mouseover="itemSelect(selected)"
+             @mouseleave="itemDeselect({delay: 250})"
+             v-if="!!dropdown.visible && selected">
```

Comment:
Sure! The difference shouldn't be noticable, unless the client is a JS-enabled potato 🥔

Reply:
Haha okay, I'll defer to you on that one. I was always under the impression it was more performant and economical to avoid adding/removing elements repeatedly in the DOM. 🤷‍♂ 
Also I was looking at the way Stripe do it, and they have an element for each dropdown menu already hidden in the DOM.